src/meteodatalab/operators/vertical_extrapolation.py

```diff
@@ -120,6 +120,47 @@ def extrapolate_geopotential_sfc2p(
     return res
 
 
+def extrapolate_k2p(
+    field: xr.DataArray,
+    p_target: float,
+) -> xr.DataArray:
+    """Extrapolate a field using constant extrapolation.
+
+    Implements the algorithm described in [1]_. For all variables except
+    temperature and geopotential (see ``extrapolate_temperature_sfc2p`` and
```

Comment:
Unsure if this will be interpreted as reference to the other functions by sphinx.